index.d.ts

```diff
@@ -0,0 +1,39 @@
+export interface AuthConfiguration extends BaseAuthConfiguration {
+    scopes: string[];
+    redirectUrl: string;
+    aditionalParameters?: {[name: string]: string};
+  }
+
+  export interface BaseAuthConfiguration{
+    issuer: string;
+    clientId: string;
+  }
+
+  export interface RevokeConfiguration{
```

Comment:
can we make sure that there's a space before `{`?

@kadikraman: Can you point prettier—I think it's set up with lint-staged, right?—at all TS files as well?